### PR TITLE
Feature/move to shared model smallfixes

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
### Description of the Change
Remove unused includes and forward declarations.

### Benefits
no unused code

### Possible Drawbacks 
no

### Usage Examples or Tests *[optional]*
no

### Alternate Designs *[optional]*
no

### CODE REVIEW

File: irohad/ametsuchi/impl/postgres_wsv_command.hpp [hunk 1]
```diff
@@ -26,15 +26,6 @@
 #include "ametsuchi/impl/postgres_wsv_common.hpp"
```
Comment (victordrobny) [inline irohad/ametsuchi/impl/postgres_wsv_command.hpp]: Rearrange includes

File: irohad/ametsuchi/impl/temporary_wsv_impl.cpp [hunk 2]
```diff
@@ -19,8 +19,6 @@
 #include "ametsuchi/impl/postgres_wsv_command.hpp"
```
Comment (victordrobny) [inline irohad/ametsuchi/impl/temporary_wsv_impl.cpp]: rearrange includes

File: irohad/consensus/yac/impl/yac_gate_impl.cpp [hunk 3]
```diff
@@ -17,7 +17,6 @@
 
 #include "consensus/yac/impl/yac_gate_impl.hpp"
```
Comment (victordrobny) [inline irohad/consensus/yac/impl/yac_gate_impl.cpp]: Add new line after this line

File: irohad/execution/impl/common_executor.cpp [hunk 4]
```diff
@@ -16,6 +16,7 @@
  */
 
 #include <algorithm>
+
 #include "common/types.hpp"
 #include "execution/common_executor.hpp"
```
Comment (victordrobny) [inline irohad/execution/impl/common_executor.cpp]: move it into beginning
Please, check https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

File: irohad/torii/processor/impl/transaction_processor_impl.cpp [hunk 5]
```diff
@@ -23,7 +23,6 @@
 namespace iroha {
```
Comment (victordrobny) [inline irohad/torii/processor/impl/transaction_processor_impl.cpp]: Same

File: irohad/validation/impl/stateful_validator_impl.cpp [hunk 6]
```diff
@@ -22,7 +22,6 @@
 
 #include "builders/protobuf/proposal.hpp"
 #include "datetime/time.hpp"
-#include "model/account.hpp"
 #include "validation/impl/stateful_validator_impl.hpp"
```
Comment (victordrobny) [inline irohad/validation/impl/stateful_validator_impl.cpp]: Same

File: irohad/ametsuchi/impl/postgres_wsv_command.hpp (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply irohad/ametsuchi/impl/postgres_wsv_command.hpp]: includes satisfy https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
1. related header
2. blank line
3. system libs
4. blank line
5. other

File: irohad/ametsuchi/impl/temporary_wsv_impl.cpp (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply irohad/ametsuchi/impl/temporary_wsv_impl.cpp]: includes already in the alphabetical order

File: irohad/torii/processor/impl/transaction_processor_impl.cpp (hunk 5 above)
Comment (Alexey-N-Chernyshov) [reply irohad/torii/processor/impl/transaction_processor_impl.cpp]: done

File: irohad/consensus/yac/impl/yac_gate_impl.cpp (hunk 3 above)
Comment (Alexey-N-Chernyshov) [reply irohad/consensus/yac/impl/yac_gate_impl.cpp]: done

**Pull request merged**